executorlib/interactive/executor.py

```
@@ -1,112 +1,274 @@
+    draw,
+    generate_nodes_and_edges,
+    generate_task_hash,
 )
-from executorlib.shared.spawner import BaseSpawner, MpiExecSpawner
-from executorlib.shared.thread import RaisingThread
+from executorlib.standalone.thread import RaisingThread
 
+try:  # The PyFluxExecutor requires flux-base to be installed.
+    from executorlib.interactive.flux import FluxPythonSpawner
+except ImportError:
+    pass
```

Ruff

Use `contextlib.suppress(ImportError)` instead of `try`-`except`-`pass`

(SIM105)